### ISSUE TITLE
Upload failed

### ISSUE THREAD

**Issue by xApep** [terminal42/contao-fineuploader#52]
Hi.
I've made a custom extension "catalog" for image upload I would like to use fineuploader.

When I upload image I get an error "Upload failed".
![image](https://user-images.githubusercontent.com/1315340/46423682-636cbf00-c737-11e8-8bf8-8f3ee7aa377c.png)

And console error.
![image](https://user-images.githubusercontent.com/1315340/46423782-aaf34b00-c737-11e8-95db-1d85ddb32d90.png)

The images are uploaded to system/tmp. 

Field configuration
```
            'label'                   => &$GLOBALS['TL_LANG']['tl_table']['myupload'],
            'exclude'                 => true,
            'inputType'               => 'fineUploader',
            'eval'                    => array
            (
                'storeFile'         => true,                // Mandatory to store the file on the server
                'multiple'          => true,                // Allow multiple files to be uploaded
                'uploadFolder'      => 'files/tmp',         // Upload target directory (can also be a Contao file system UUID)
                'useHomeDir'        => true,                // Upload to the FE member home directory (overrides "uploadFolder", can also be a Contao file system UUID)
                'uploaderConfig'    => "['debug': true]",   // Custom uploader configuration that gets merged with the other params
                'uploaderLimit'     => 4,                   // Maximum files that can be uploaded
                'addToDbafs'        => true,                // Add files to the database assisted file system
                'extensions'        => 'jpg,png,gif',           // Allowed extension types
                'minlength'         => 1048,                    // Minimum file size
                'maxlength'         => 204800000,             // Maximum file size (is ignored if you use chunking!)
                'maxWidth'          => 8000,                 // Maximum width (applies to images only)
                'maxHeight'         => 6000,                 // Maximum height (applies to images only)
                'doNotOverwrite'    => true,                // Do not overwrite files in destination folder
                'uploadButtonLabel' => 'Upload images',     // Custom upload button label

                'chunking'          => false,                // Enable chunking
                'chunkSize'         => 2000000,             // Chunk size in bytes
                'concurrent'        => true,                 // Allow multiple chunks to be uploaded simultaneously per file
                'maxConnections'    => 3,                    // Maximum allowable concurrent requests
                'directUpload'      => false,
            ),
            'sql'                     => "blob NULL"
```

I'm using Contao 3.5.36.

**Comment by aschempp** [comment]
@qzminski any ideas?

**Comment by qzminski** [comment]
Can you check the system logs located in `system/logs/error.log` and see if there's anything useful there?

**Comment by rflx** [comment]
Same issue with Contao 4.6.8 on a local environment working on Apache with PHP 7.2.
There is also a ticket on FineUploaders Github repository: https://github.com/FineUploader/fine-uploader/issues/1558 with a potential [solution](https://github.com/FineUploader/fine-uploader/issues/1558#issuecomment-307713855) for this issue in `jquery.fine-uploader.js`.

Furthermore it seems that they revoked the development of FineUploader?

@qzminski this is my error log:
```
[2018-11-26 10:23:45] request.INFO: Matched route "contao_frontend". {"route":"contao_frontend","route_parameters":{"_route":"contao_frontend","_token_check":true,"_controller":"Contao\\CoreBundle\\Controller\\FrontendController::indexAction","_scope":"frontend","alias":"daten-upload"},"request_uri":"http://domain:8888/daten-upload.html","method":"POST"} []
[2018-11-26 10:23:45] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2018-11-26 10:23:45] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Compile Error: Cannot make static method Contao\FileUpload::getMaximumUploadSize() non static in class Haste\Util\FileUpload" at /Applications/MAMP/htdocs/domain/vendor/codefog/contao-haste/library/Haste/Util/FileUpload.php line 5 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Compile Error: Cannot make static method Contao\\FileUpload::getMaximumUploadSize() non static in class Haste\\Util\\FileUpload at /Applications/MAMP/htdocs/domain/vendor/codefog/contao-haste/library/Haste/Util/FileUpload.php:5)"} []
```
and the console output:
![errorfineuploader](https://user-images.githubusercontent.com/7251106/49007322-79ad5f00-f16b-11e8-8cf2-2d577147a9b9.png)

**Comment by rflx** [comment]
Update to Contao 4.6.8 fixed my issue (see contao [#188](https://github.com/contao/contao/pull/188)).
